### ISSUE TITLE
media manager doesn't start on early media

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#134]
```
What steps will reproduce the problem?
1. Make early media session start
2. Wait for video is useless because no video prepare/start is done

What is the expected output? What do you see instead?
Video starts before having answered the call

What version of the product are you using? On what operating system?
latest


Actually it should be a doubango issue

```

Original issue reported on code.google.com by `sylar1...@gmail.com` on 21 Dec 2010 at 11:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 24 Dec 2010 at 12:33
- Changed state: **NextRelease**

**Comment by GoogleCodeExporter** [comment]
```
is there any news about this issue? I really need it asap :(
```

Original comment by `sylar1...@gmail.com` on 11 Jan 2011 at 8:54

**Comment by GoogleCodeExporter** [comment]
```
I need this feature ASAP too. Is there a way you can point me to the right 
place in the code and give me at least a rough idea how to fix it? Thank you.
```

Original comment by `tayurs...@gmail.com` on 5 Jul 2011 at 1:17

**Comment by GoogleCodeExporter** [comment]
```
Fixed by IMSDroid v2.0.431
```

Original comment by `boss...@yahoo.fr` on 14 Jul 2011 at 5:55
- Changed state: **Fixed**
